### DISCUSSION TITLE
Adding value mappings to the heatmap panel

### DISCUSSION THREAD

**Discussion by trylaarsdam** [grafana/grafana#70718] [Dashboards, Panels & Visualizations]
Would it be possible for value mappings to be added to the heatmap panel? They are present in the histogram and timeseries panels, but not in the heatmap.
For instance, in my histogram I can map the various sleep states we measure to their actual labels:
<img width="1440" alt="Screenshot 2023-06-26 at 1 46 11 PM" src="https://github.com/grafana/grafana/assets/50586094/6cb36fb0-3683-48e9-a449-c1100ea7aab8">
<img width="351" alt="Screenshot 2023-06-26 at 1 46 30 PM" src="https://github.com/grafana/grafana/assets/50586094/e03721f8-cb07-4f75-a13e-3aabbae4e72b">
But in the heatmap this feature is not present.

Thanks,
Todd Rylaarsdam

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82588. Feel free to continue the discussion around this there. Thank you!
